script/release.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+const { versionBump, currentVersion }  = require('./version-utils');
+const { logger, spawnOrFail, prompt, shouldContinuePrompt, quit, process, path } = require('./cli-utils');
+
+const deployDemo = (version) => {
+  const demo_name = `chime-sdk-demo-${version.replace(/\./g, "-")}`;
+  logger.log(`Deploying ${demo_name} ...`);
+  process.chdir(path.join(__dirname, '../demos/serverless'));
+  spawnOrFail('npm', [`run deploy -- -b ${demo_name} -s ${demo_name} -o ${demo_name} -u false`], { printErr: true });
+};
+
+const getCurrentRemoteBranch = () => {
+  return (spawnOrFail('git', ['for-each-ref --format="%(upstream:short)" "$(git symbolic-ref -q HEAD)"'], { skipOutput: true })).trim();
+};
+
+const buildAndPack = () => {
+  logger.log('Building package...');
+  spawnOrFail('npm', ['run build:release']);
+  logger.log('Packaging ...');
+  spawnOrFail('npm', ['pack --dry-run'], { printErr: true });
+};
+
+const cleanUp = async (remoteBranch) => {
```

Comment:
It seems like the main purpose of this method is to discard any pending changes in the local branch. If so, can you rename it to something like `cleanUpPendingChanges`?

I just want the method name to a bit more precise. cleanUp as a name is too broad as it might indicate linting, discarding pending changes, removing dist folders etc.

Reply:
It can be any clean up in the repo before release or pending changes is a bit too specific.

Reply:
Ok so you want to keep the name generic for now. In future, we might add some more clean up before release as well?

Sounds good then.

Reply:
`resetBranch`?